documentation/general/dash-sonic-hld.md

```diff
@@ -213,14 +213,16 @@ The following section captures at a high-level on the VNET packet flow. Detailed
 
 ## 2.1 Outbound packet processing pipeline
 	
-  ![dash-outbound](./images/dash-hld-outbound-packet-processing-pipeline.png)
+  ![dash-outbound](./images/dash-hld-outbound-packet-processing-pipeline.svg)
 	
 Based on the incoming packet's VNI matched against the reserved VNI assigned for VM->Appliance, the pipeline shall set the direction as TX(Outbound) and using the inner src-mac, maps to the corresponding ENI.The incoming packet will always be VXLAN encapsulated and outer dst-ip is the appliance VIP. The pipeline shall parse the VNI, and for VM traffic, the VNI shall be a special reserved VNI. Everything else shall be treated as as network traffic(RX). Pipeline shall use VNI to differentiate the traffic to be VM (Inbound) or Network (Outbound).
 
 In the outbound flow, the appliance shall assume it is the first appliance to apply policy. It applies the outbound ACLs in three stages (VNIC, Subnet and VNET), processed in order, with the outcome being the most restrictive of the three ACLs combined. 
 
 After the ACL stage, it does LPM routing based on the inner dst-ip and applies the respective action (encap, subsequent CA-PA mapping). Finally, update the connection tracking table for both inbound and outbound.
 
+After the connection tracking update, the packet proceeds to the "underlay routing" stage. Based on the destination IP address (outer dst-ip), the underlay routing block decides how the packet should be forwarded towards its destination.
+
```

Comment:
This statement seems to imply that a lookup on outer_dst_ip is mandatory, but the HLD mentions that an LPM table for underlay is strictly optional.

https://github.com/sonic-net/DASH/blob/main/documentation/general/dash-sonic-hld.md#:~:text=Note%20that%20only%20default%20route%20is%20expected%20from%20the%20peer%20BGP%20and%20appliance%20is%20not%20expected%20to%20allocate%20an%20LPM%20resource%20for%20underlay.

I understand that the underlay_routing table in the underlay.p4 program has a default action of send_to_input_port, and the program doesn't require the underlay routes to be programmed. But this statement probably can be worded better to clarify that dst-ip based underlay routing lookup is an optional step.